src/CMakeLists.txt

```diff
@@ -46,56 +39,50 @@ add_subdirectory(imgui)
 add_subdirectory(resource)
 add_subdirectory(asm)
 
-if(ENABLE_CEMUHOOK)
-    add_definitions(-DUSE_CEMUHOOK)
-    add_subdirectory(cemuhook)
-endif()
-
-if(PUBLIC_RELEASE)
-add_executable(CemuBin WIN32
-main.cpp
-mainLLE.cpp
-)
-else()
-add_executable(CemuBin
-main.cpp
-mainLLE.cpp
-)
+if (ENABLE_CEMUHOOK)
+  add_definitions(-DUSE_CEMUHOOK)
+  add_subdirectory(cemuhook)
 endif()
 
-target_precompile_headers(CemuBin PRIVATE Common/precompiled.h)
+add_executable(Cemu main.cpp mainLLE.cpp)
 
-if(WIN32)
-  target_sources(CemuBin PRIVATE
-  resource/cemu.rc
-  exports.def # for Cemuhook
+if (WIN32)
+  target_sources(Cemu PRIVATE
+    resource/cemu.rc
+    exports.def # for Cemuhook
   )
+  set_target_properties(Cemu PROPERTIES WIN32_EXECUTABLE TRUE)
 endif()
 
-set_property(TARGET CemuBin PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
-
-set_target_properties(CemuBin PROPERTIES
-        RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_CURRENT_SOURCE_DIR}/../bin/
-        RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_CURRENT_SOURCE_DIR}/../bin/
-        OUTPUT_NAME "Cemu"
-        )
-
-target_link_libraries(CemuBin PRIVATE CemuCommon CemuComponents CemuCafe CemuConfig CemuGui CemuAudio CemuInput CemuUtil)
-target_link_libraries(CemuBin PRIVATE CemuAsm)
-target_link_libraries(CemuBin PRIVATE OpenSSL::SSL)
-target_link_libraries(CemuBin PRIVATE ZLIB::ZLIB)
-target_link_libraries(CemuBin PRIVATE ${wxWidgets_LIBRARIES})
-target_link_libraries(CemuBin PRIVATE CURL::libcurl)
-target_link_libraries(CemuBin PRIVATE imgui::imgui)
-target_link_libraries(CemuBin PRIVATE pugixml pugixml::static pugixml::pugixml)
+set_target_properties(Cemu PROPERTIES OUTPUT_NAME "Cemu")
+
+target_link_libraries(Cemu PRIVATE
+  Common
+  Components
+  Cafe
+  Config
+  Gui
+  Audio
+  Input
+  Util
+  Assembly
+  OpenSSL::SSL
+  ZLIB::ZLIB
+  ${wxWidgets_LIBRARIES}
```

Comment:
Linking with the libraries is not enough. You should also use the `wxWidgets_INCLUDE_DIRS`, `wxWidgets_DEFINITIONS`, `wxWidgets_CXX_FLAGS`, `wxWidgets_LIBRARY_DIRS`, etc.